modin/core/execution/ray/implementations/cudf_on_ray/io/text/csv_dispatcher.py

```diff
@@ -52,7 +49,7 @@ def build_partition(cls, partition_ids, row_lengths, column_widths):
 
         def create_partition(i, j):
             return cls.frame_partition_cls(
-                GPU_MANAGERS[i],
+                cls.frame_partition_mgr_cls._get_gpu_managers()[i],
```

Comment:
```suggestion
                cls.frame_partition_mgr_cls.get_gpu_managers()[i],
```
Should probably be public.

Reply:
not actual